deployments/generic/kubearmor.yaml

```
@@ -190,6 +196,10 @@ spec:
           path: /etc/os-release
           type: File
         name: os-release-path
+      - hostPath:
+          path: /boot
+          type: DirectoryOrCreate
```

```suggestion
          type: Directory
```
should not create the directory.